packages/docs/components/page-configs/va-carousel/api-options.ts

```diff
@@ -0,0 +1,27 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
draggable is kinda easy to break
![2020-10-20_16-05-03](https://user-images.githubusercontent.com/5394573/96589906-0fb57a00-12ee-11eb-96a9-5b84976642ee.gif)